### PR TITLE
[boinc] Update to 7.20.0

### PR DESCRIPTION
**Pull request by AenBleidd**
Signed-off-by: Vitalii Koshura <lestat.de.lionkur@gmail.com>

### CODE REVIEW

File: versions/baseline.json [hunk 1]
```diff
@@ -2742,7 +2742,7 @@
     },
     "hareflow": {
       "baseline": "0.1.0",
-      "port-version": 0      
+      "port-version": 0
```
Comment (FrankXie05) [inline versions/baseline.json:2745]:
```
      "port-version": 0
```
Why modify unrelated ports？

**Review by FrankXie05 — CHANGES_REQUESTED** [review]

File: versions/baseline.json (hunk 1 above)
Comment (dg0yt) [reply versions/baseline.json:2745]: This is from `vcpkg x-add-version ...`. It shouldn't create a merge conflict.

**Comment by FrankXie05** [comment]
The patch has been merged upstream.

**Review by FrankXie05 — APPROVED** [review]

**Comment by ras0219-msft** [comment]
Thanks for the update!

**Pull request merged**